easybuild/easyconfigs/o/OpenFOAM/OpenFOAM-v1612+-foss-2016a.eb

```diff
@@ -0,0 +1,46 @@
+# -*- python -*-
+name = 'OpenFOAM'
+version = 'v1612+'
+
+homepage = 'http://www.openfoam.com/'
+description = """OpenFOAM is a GPL-opensource C++ CFD-toolbox.
+This offering is not approved or endorsed by OpenCFD Ltd,
+producer and distributor of the OpenFOAM software via www.openfoam.com,
+and owner of the OPENFOAM trademark.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = [
+    ('https://sourceforge.net/projects/openfoamplus/files/%(version)s',
+     'download')
+]
+
+sources = [ 'OpenFOAM-%(version)s.tgz' ]
+checksums = ['ca02c491369150ab127cbb88ec60fbdf']
+
+patches = [ 'OpenFOAM-%(version)s-cleanup.patch' ]
+
+dependencies = [
+    # > causes bash issue? ('libreadline', '6.3'),
+    # > causes bash issue? ('ncurses', '6.0'),
```

Comment:
I experienced an issue where the LD_LIBRARY_PATH pointed to the easybuild libreadline, which provoked symbol conflicts for my bash.